massgov/openmass
# DP-32091: Modify session_orgs to include org node ids

**Description:**
See ticket


**Jira:** (Skip unless you are MA staff)
DP-32091


**To Test:**
 - Navigate to one or more pages with related orgs, such as `/orgs/executive-office-of-elder-affairs` 
 - Open the console
 - Go to the application tab
 - View the value for the `session_orgs` key under session storage
 - It should be in the format of `firstOrg:123,secondOrg:124`


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
